code/datums/wounds/types/special.dm

```diff
@@ -94,6 +94,11 @@
 /datum/wound/facial/eyes/right/permanent
 	whp = null
 	woundpain = 0
+	sound_effect = null
+
+/datum/wound/facial/eyes/right/permanent/on_mob_gain(mob/living/affected)
+	if(mob_overlay)
+		affected.update_damage_overlays()
```

Comment:
Shouldn't this change be added to a parent type?

Reply:
This looks vaguely like it should be handled on /datum/wound or something, I'm not sure what makes sense here!